src/Console/Command/CleanMySQLDatabaseCommand.php

```diff
@@ -0,0 +1,177 @@
+<?php
+declare(strict_types=1);
+
+namespace duckpony\Console\Command;
+
+use JiraRestApi\Configuration\ArrayConfiguration;
+use JiraRestApi\Issue\IssueService;
+use JiraRestApi\JiraException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Class CleanMySQLDatabaseCommand
+ *
+ * @package duckpony\Console\Command
+ * @author  Michel Petiton <michel.petiton@check24.de>
+ */
+class CleanMySQLDatabaseCommand extends AbstractCommand
+{
+    private const DATABASE_BLACKLIST = [
+        'information_schema',
+        'perfomance_schema',
+        'mysql',
+    ];
+
+    /**
+     * Configures the console command
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        $this->addOption('status', 's', InputOption::VALUE_REQUIRED, 'Status');
+        $this->addOption('pattern', 'p', InputOption::VALUE_REQUIRED, 'Branch pattern');
+        $this->addOption('invert', 'i', InputOption::VALUE_NONE, 'Invert status');
+        $this->addOption('config', 'c', InputOption::VALUE_REQUIRED, 'Config', $this->getRootPath() . '/config/config.yml');
+        $this->addArgument('branchname-filter', InputArgument::IS_ARRAY | InputArgument::OPTIONAL , 'Filter branchname');
+
+        $this->setName('db:clean')
+             ->setDescription('Scans Database and cleans orphaned')
+             ->setHelp(
+                 <<<EOT
+Scans MySQL Databases and removes
+them under certain conditions
+EOT
+             );
+    }
+
+    /**
+     * Executes the console command
+     *
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return int|null|void
+     * @throws \JiraRestApi\JiraException
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
```

Comment:
Since you only  return `int` (if you change the one `null` return) you can have `: int` return type here